app/src/main/java/com/sschuraytz/charcoaldrawing/DrawingView.java

```diff
@@ -116,6 +116,35 @@ public void createNewCanvas() {
         invalidate();
     }
 
+    public void lighter() {
+        //alpha must be at least 0
+        //since alpha of each tool is set to the difference of opacity & radius, ensure opacity is at least the size of max-radius (100)
+        if (currentTool.getOpacity() >= 125) {
+            showToast("lighter");
+            showToast(currentTool.getOpacity() + " ");
+            currentTool.updateOpacity(-OPACITY_INCREMENTER);
```

Comment:
"update" is a bit misleading here. It's very easy to confuse someone and think it's a "set" method.
You are incrementing it.